app/presenters/setting_presenter.rb

```diff
@@ -15,20 +15,24 @@ class SettingPresenter
   attribute :settings_type, :string
   attribute :select_values
   attribute :config_file
+  attribute :updated_at, :datetime
+  attribute :created_at
```

Comment:
Why is one `:datetime` and the other isn't?

Reply:
I was thinking about having `nil` or `:not_changed` as a special value for this

Reply:
But that's something for the follow-up I guess :))